### PR TITLE
New new branch

### PR DESCRIPTION
**Pull request by milana986**

### CODE REVIEW

File: src/main/webapp/static/app/components/core/navbar.html [hunk 1]
```diff
@@ -2,7 +2,9 @@
   <div class="container-fluid">
     <ul class="nav navbar-nav">
       <li class="icon-logo"></li>
-      <li><p style="color: whitesmoke; font-weight: bold; padding: 1.4em 0.5em; font-size: 20px; font-family: 'Roboto', sans-serif;">NEWS PORTAL</p></li>
+      <li><p
+        style="color: whitesmoke; font-weight: bold; padding: 1.4em 0.5em; font-size: 20px; font-family: 'Roboto', sans-serif;">
```
Comment (DraganU) [inline src/main/webapp/static/app/components/core/navbar.html:6]: Please fix this formatting... "p" tag should be in one line.

File: src/main/webapp/static/app/components/news/news.controller.js [hunk 2]
```diff
@@ -19,9 +34,9 @@
     $scope.removeNews = function(vest) {
       NewsService.deleteNews(vest)
           .then($timeout(function() {
-            $state.reload();  //dovoljno je samo ovo (umesto stateRefresh();) a sa pozivom $state.go('main.news') NECE RADIti
-          }, 300)             //JER JE ON VEC NA NEWS STR. NE MOZE DA SE OPET PREKO 'main.news', vrati na samu sebe neg mora state.reload
-      )
+                $state.reload();  //dovoljno je samo ovo (umesto stateRefresh();) a sa pozivom $state.go('main.news') NECE RADIti
```
Comment (DraganU) [inline src/main/webapp/static/app/components/news/news.controller.js:36]: fix this wrong indentation.

Comment (milana986) [reply src/main/webapp/static/app/components/news/news.controller.js:36]: i can't change it. Identation is  setted automatically.

File: src/main/webapp/static/app/components/core/navbar.html (hunk 1 above)
Comment (milana986) [reply src/main/webapp/static/app/components/core/navbar.html:6]: fixed.

**Pull request merged**